conf/messages

```diff
@@ -53,7 +53,8 @@ lsp.onThreshold.p1 = You have reached the financial penalty threshold.
 agent.lsp.onThreshold.p1 = Your client has reached the financial penalty threshold.
 lsp.onThreshold.p2 = This means you have to pay a £{0} penalty every time you submit a VAT Return late, until we remove your penalty points.
 agent.lsp.onThreshold.p2 = This means they have to pay a £{0} penalty every time they submit a VAT Return late, until we remove their points.
-lsp.onThreshold.link = Actions to take before HMRC can remove your points
+lsp.onThreshold.p3 = Earliest dates your points could be removed:
+lsp.onThreshold.link = Actions to take to get your points removed by {0}
```

Comment:
the message would be 'Actions to take to get your points removed' instead, 'by {0}' could be deleted.

Reply:
My bad! the message is consistent with the content master v2.